test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/amendment/group/GroupMembersListControllerSpec.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.controllers.amendment.group
+
+import base.PptTestData.newUser
+import org.mockito.ArgumentMatchers.{any, refEq}
+import org.mockito.Mockito.{verify, when}
+import play.api.http.Status.{BAD_REQUEST, OK}
+import play.api.mvc.{AnyContent, Request, Result}
+import play.api.test.CSRFTokenHelper.CSRFRequest
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{
+  contentAsString,
+  redirectLocation,
+  status,
+  stubMessagesControllerComponents
+}
+import play.twirl.api.Html
+import uk.gov.hmrc.plasticpackagingtax.registration.controllers.group
+import uk.gov.hmrc.plasticpackagingtax.registration.views.html.amendment.group.list_group_members_page
+
+import scala.concurrent.Future
+
+class GroupMembersListControllerSpec extends AmendGroupControllerSpec {
```

Comment:
For consistency with existing code, I think we should refactor to use `MockAuthAction` and `MockAmendmentJourneyAction` traits.

Reply:
:+1: